pkgs/development/interpreters/python/cpython/3.5/no-ldconfig.patch

```diff
@@ -12,53 +12,35 @@ diff --git a/Lib/ctypes/util.py b/Lib/ctypes/util.py
 index 7684eab81d..e9957d7951 100644
 --- a/Lib/ctypes/util.py
 +++ b/Lib/ctypes/util.py
-@@ -95,46 +95,7 @@ elif os.name == "posix":
+@@ -94,28 +94,7 @@ elif os.name == "posix":
      import re, tempfile
- 
+
      def _findLib_gcc(name):
--        # Run GCC's linker with the -t (aka --trace) option and examine the
--        # library name it prints out. The GCC command will fail because we
--        # haven't supplied a proper program with main(), but that does not
--        # matter.
--        expr = os.fsencode(r'[^\(\)\s]*lib%s\.[^\(\)\s]*' % re.escape(name))
--
--        c_compiler = shutil.which('gcc')
--        if not c_compiler:
--            c_compiler = shutil.which('cc')
--        if not c_compiler:
--            # No C compiler available, give up
--            return None
--
--        temp = tempfile.NamedTemporaryFile()
+-        expr = r'[^\(\)\s]*lib%s\.[^\(\)\s]*' % re.escape(name)
```

Comment:
This changed between 3.5.2 and 3.5.9. I downloaded 3.5.2 locally and created this git patch manually. Seems to be the only difference.